Libraries/Components/CheckBox/AndroidCheckBoxNativeComponent.js

```diff
@@ -47,7 +51,12 @@ type NativeProps = $ReadOnly<{|
 
   on?: ?boolean,
   enabled?: boolean,
-  tintColors: {|true: ?number, false: ?number|} | typeof undefined,
+  tintColors:
```

Comment:
Why do these take `ColorValue` as well as `ProcessedColorValue`? I'd expect them to only take `ProcessedColorValue`.

Reply:
Yes, I agree, changed and Flow check still works.   I think that when I originally wrote this in our fork, these were `number | string` but have since changed to just `number`.